images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -11,6 +11,9 @@ sleep "${APPLICATION_INIT_DELAY}"
 # Prepare initialization environment
 prepare_init_env
 
+# Generate the certs if needed
+${APPLIANCE_ROOT}/initialize_appliance.sh
+
```

Comment:
I think there is a race on this one, might cause a new v2_key to be generated on every pod re-schedule. The image comes baked without keys/certs, we restore the original backed up certs/keys from the PV on every re-schedule. This happens [here](https://github.com/ManageIQ/manageiq-pods/blob/master/images/miq-app/docker-assets/container-scripts/container-deploy-common.sh#L281)